internal/cephfs/nodeserver.go

```diff
@@ -422,3 +509,258 @@ func (ns *NodeServer) NodeGetVolumeStats(
 
 	return nil, status.Errorf(codes.InvalidArgument, "targetpath %q is not a directory or device", targetPath)
 }
+
```

Comment:
move below changes to a new file fuse-recovery.go?

Reply:
Sure, unfortunately it will probably need to stay in the same package because it's using some unexported functions? Refactoring that would be messy I think.